profile.go

```diff
@@ -131,6 +132,29 @@ Make sure to backup your data frequently.`,
 			return nil
 		},
 	},
+	"postgresds": {
+		Description: `Replaces default datastore configuration with experimental
+postgres datastore.
+
+WARNING: postgres datastore is experimental.
+Make sure to backup your data frequently.`,
+		Transform: func(c *Config) error {
+			c.Datastore.Spec = map[string]interface{}{
+				"type":   "measure",
+				"prefix": "postgres.datastore",
+				"child": map[string]interface{}{
+					"type":     "postgres",
+					"host":     os.Getenv("IPFS_PGHOST"),
+					"user":     os.Getenv("IPFS_PGUSER"),
+					"passfile": os.Getenv("IPFS_PGPASSFILE"),
```

Comment:
`go fmt ./...`

Reply:
I'm not seeing any changes when I go fmt ./... locally

Reply:
Hmm.. Probably github just being weird